### ISSUE TITLE
Staging Review finding: File upload does not convey status to screen readers

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122412]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Design System & Forms Library
**Product:** Form 21-4140
**Feature:** Employment Questionnaire

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When a file is being uploaded, a progress animation is visible, however this information is not conveyed programmatically, which means the screen reader users are not receiving a confirmation that something is happening after they took the upload action. 

<details><summary>Link, screenshot or steps to recreate</summary>

- Using a screen reader
- Navigate to step 5 of 6 Upload supporting evidence
- TAB to the file upload and activate using Enter
- Select a file in the system dialog and click "Open" or equivalent
- Note screen reader announcement

</details>

## Recommended action

Provide a screen reader indication that the upload is occurring, especially for items that take longer than several seconds.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
Should be resolved in this issue - https://github.com/department-of-veterans-affairs/component-library/pull/1886#event-22025596942
